### DISCUSSION TITLE
Command to list all tasks (in my Makefile.toml)?

### DISCUSSION THREAD

**Discussion by xpe** [sagiegurari/cargo-make#1100] [Q&A] [answered]
I've looked but haven't found a command to list all tasks in my Makefile.toml. Does it exist? If so, how did I miss it?

**Comment by sagiegurari** [comment, marked as answer]
https://github.com/sagiegurari/cargo-make?tab=readme-ov-file#usage-cli

--list-all-steps and --list-category-steps

**Comment by xpe** [comment]
@sagiegurari Thanks.

I see "tasks" isn't part of that command -- that's probably why I didn't find it.

The documentation mentions "steps" without defining them. Are they the same as tasks? (My guess would be that "steps" and "tasks" are implicit synonyms but never defined as such.)
